### DISCUSSION TITLE
Are we considering any alternatives for @ExperimentalTime

### DISCUSSION THREAD

**Discussion by nehalsep7** [dropbox/Store#487] [General]
Wiki doesn't suggest using  @ExperimentalTime annotation for libraries. https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.time/-experimental-time/ I heard from folks that we plan to move away from this annotation but still see it being used.

**Comment by matt-ramotar** [comment]
Hey! Thanks for reaching out. We do not use it in [Store5](https://github.com/MobileNativeFoundation/Store/tree/store5/store/src/commonTest/kotlin/org/mobilenativefoundation/store/store5). Currently we are migrating Store4 testing logic (#449). But as far as I know we don't currently have any plans to re-write the Store4 tests. Thoughts?

**Reply by nehalsep7** [reply]
thanks for the reply Matt! if its already unused in Store 5, that sounds good.
